addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1299,6 +1299,9 @@ var SnippetEditor = Widget.extend({
         self.draggableComponent.$scrollTarget.on('scroll.scrolling_element', function () {
             self.$el.trigger('scroll');
         });
+
+        this.onParentWindowMouseup = this._onParentWindowMouseup.bind(this);
+        window.parent.addEventListener('mouseup', this.onParentWindowMouseup, {once: true});
```

Comment:
I double-checked because it still felt weird: `window` is already the top window in the case of the website builder, so I don't think `window.parent` ever make sense here :grimacing: